docs/api/breaking-changes.md

```diff
@@ -6,6 +6,17 @@ Breaking changes will be documented here, and deprecation warnings added to JS c
 
 The `FIXME` string is used in code comments to denote things that should be fixed for future releases. See https://github.com/electron/electron/search?q=fixme
 
+## Planned Breaking API Changes (7.0)
+
+The following deprecated APIs are being removed:
+- `powerMonitor.querySystemIdleState()`
+- `powerMonitor.querySystemIdleTime()`
+- `webFrame.setIsolatedWorldContentSecurityPolicy()`
+- `webFrame.setIsolatedWorldHumanReadableName()`
+- `webFrame.setIsolatedWorldSecurityOrigin()`
+- `app.getApplicationMenu()`
+- `app.setApplicationMenu()`
```

Comment:
These need to follow the format of all other methods in this document; for example
```
- `app.getApplicationMenu()`
- `app.setApplicationMenu()`
```
are being replaced with a property `app.applicationMenu` which should be made clear to users without their needing to dig through other components of our docs

Reply:
The `powerMonitor` and `webFrame` methods are already documented as deprecated in Electron 5 and 6. The `app` were not documented at all before. That's why I was wondering why I was asked to add it to "Planned Breaking Changes". In past we didn't explicitly mention removed methods after deprecation again in the document.